src/test/java/teammates/test/cases/BaseTestCaseWithDatastoreAccess.java

```diff
@@ -199,8 +200,10 @@ protected CourseAttributes getCourse(CourseAttributes course) {
     }
 
     private void equalizeIrrelevantData(CourseAttributes expected, CourseAttributes actual) {
-        // Ignore time field as it is stamped at the time of creation in testing
+        // ignore time field as it is stamped at the time of creation in testing
         expected.createdAt = actual.createdAt;
+        // sanitize fields
+        expected.sanitizeForSaving();
```

Comment:
This change is required due to insertion of non-sanitized data in typicalDataBundle. Unrelated tests will fail without sanitizing the original data first.